vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/transformer/HeadlessCrawlerTableTransformer.java

```diff
@@ -66,7 +68,24 @@ private void addSeeds(URI mainApplicationPage, CrawlController controller)
                 .map(mainApplicationPagePath::concat)
                 .map(relativeUrl -> UriUtils.buildNewUrl(mainApplicationPage, relativeUrl))
                 .map(URI::toString)
-                .forEach(controller::addSeed);
+                .forEach(pageUrl -> addSeed(controller, pageUrl));
+    }
+
+    private void addSeed(CrawlController controller, String pageUrl)
+    {
+        try
+        {
+            controller.addSeed(pageUrl);
+        }
+        catch (IOException e)
+        {
+            throw new UncheckedIOException(e);
+        }
+        catch (InterruptedException e)
```

Comment:
is this exception thrown by purpose? maybe we can propose improvements to avoid this handling

Reply:
These changes come from the unreleased origin/master branch, https://github.com/yasserg/crawler4j/commit/182971b9a37e7abae5ae5553f4a6fd41bade985b#diff-0e31ac1ca16f06e65a5d60e843a84c828d10b3ff30312d540a314bcfc9a92e33 line 177

As per the isHaltOnError() I guess it should fail only on RuntimeExceptions since by catch clauses above checked exceptions just cached and logged, besides InterruptedException